src/main/scala/com/bwsw/cloudstack/entities/Executor.scala

```diff
@@ -29,13 +29,25 @@ import org.slf4j.LoggerFactory
 
 import scala.util.{Failure, Success, Try}
 
+/**
+  * Class is responsible for interaction with CloudStack
+  *
+  * @param settings see: Executor.Settings
+  * @param waitIfServerUnavailable If true and the server is unavailable, send requests until available.
```

Comment:
If true and the server is unavailable, sends requests until the server has become available

Reply:
ok